app/assets/stylesheets/shared/form.scss

```diff
@@ -1,5 +1,21 @@
 // material style floating form labels
 .form-floating {
+  display: flex;
+  align-items: center;
+  border: 1px solid $gray-200;
+  border-radius: 0.1875rem;
+
+  &:focus-within {
+    border-color: $blue-light;
+    box-shadow: 0 0 0 0.25rem rgba(0,114,163,0.25);
+  }
+
+  &.textarea-help {
+    .form-control {
+      padding-right: 2.5rem;
+    }
+  }
```

Comment:
textareas were tricky because of the resizing. Had to take a slightly different approach.

Reply:
We're going to replace those with WYSIWYG at some point anyways.